apps/logsupport_appdefs.cpp

```diff
@@ -12,12 +12,39 @@
 LogFANames::LogFANames()
 {
     Install("GENERAL", SRT_LOGFA_GENERAL);
-
-    Install("CONTROL", SRT_LOGFA_CONTROL);
-    Install("DATA", SRT_LOGFA_DATA);
+    Install("SOCKMGMT", SRT_LOGFA_SOCKMGMT);
+    Install("CONN", SRT_LOGFA_CONN);
+    Install("XTIMER", SRT_LOGFA_XTIMER);
     Install("TSBPD", SRT_LOGFA_TSBPD);
-    Install("REXMIT", SRT_LOGFA_REXMIT);
+    Install("RSRC", SRT_LOGFA_RSRC);
 
     Install("CONGEST", SRT_LOGFA_CONGEST);
+    Install("PFILTER", SRT_LOGFA_PFILTER);
```

Comment:
Might be helpful to have somedescription guidline and decoding of abbreviations in comments.

Reply:
Yes, but I think not in generated files.

I was thinking about putting the sole definition of the FA in a separate file, just read by the generator script. What do you think?